docs/content/en/docs/migrate/evolution-klt/_index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Evolution of KLT
+description: Understand the paradigm of KLT, how it evolved from Keptn v1, and whether you should migrate to KLT
+weight: 5
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+The Keptn products provide tool-agnostic ways
+to manage and observe software deployment.
+Keptn connects to the tools you choose
+through an open event standard
+to automate delivery and remediation sequences.
+
+Two different products are available:
+
+* **First Generation:** Keptn v1.y.z LTS, referred to as "Keptn v1"
+* **Second Generation:** Keptn Lifecycle Toolkit, referred to as KLT
+
+In this page, we give a high-level overview
+of each of these products
+to help you understand the similarities and differences
+and which is most appropriate for your environment.
+
+> **Note**
+This section is under development.
+Information that is published here has been reviewed for technical accuracy
+but the format and content is still evolving.
+We hope you will contribute your experiences
+and questions that you have.
+
+## Keptn v1 (Keptn v1.y.z LTS)
+
+Development of Keptn v1 began in 2019.
+In 2022, it reached incubation status
+and the LTS (Long-Term Support) version was released.
+Keptn v1 is designed for the classic monolithic development world,
```

Comment:
This section IMO isn't correct. This sets the tone that Keptn LTS is irrelevant to anyone on microservices. That's not true. Keptn LTS is perfectly capable of being used in a microservices architecture.

Indeed, the sentenceparagraph below alludes to this: `Keptn v1 is a general purpose orchestration engine
for cloud and non-cloud native tooling on Kubernetes.`

Reply:
Comment added as TODO to source